src/jit/lowerxarch.cpp

```diff
@@ -903,6 +903,16 @@ void Lowering::TreeNodeInfoInit(GenTree* stmt)
             }
 
             // Set destination candidates for return value of the call.
+#ifdef _TARGET_X86_
+            if ((tree->gtCall.gtCallType == CT_HELPER) && (tree->gtCall.gtCallMethHnd == compiler->eeFindHelper(CORINFO_HELP_INIT_PINVOKE_FRAME)))
```

Comment:
To make the above member function more generic to useful in more ways we could add

```
// Returns true if the call node is a call to a given helper.
bool GenTree::IsHelperCall(CorInfoHelpFunc  helper)
{
}

// usage
if (tree->IsHelperCall(CORINFO_HELP_INIT_PINVOKE_FRAME))
```